scripts/index.js

```diff
@@ -0,0 +1,110 @@
+let tempStart = 71;
```

Comment:
ID's that you are referencing here in your index.js file are not even in your html file. To get anything functional working that needs to be fixed.  For example, you are referencing the garden content but that has been established in your html file yet.